src/openqaoa-core/algorithms/rqaoa/rqaoa_workflow_properties.py

```diff
@@ -1,24 +1,13 @@
-#   Copyright 2022 Entropica Labs
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-
 from typing import Union
-from .parameters import Parameters
-from openqaoa.qaoa_parameters import Hamiltonian
 
-ALLOWED_RQAOA_TYPES = ['adaptive', 'custom']
+from ..workflow_properties import WorkflowProperties
+from ...qaoa_components import Hamiltonian
+
+
+ALLOWED_RQAOA_TYPES = ["adaptive", "custom"]
 
-class RqaoaParameters(Parameters):
+
+class RqaoaParameters(WorkflowProperties):
```

Comment:
Why this object is not in `workflow_properties.py`? 
I think in `workflow_properties.py` there should be or:
- Only the Properties Objects used in `baseworkflow.py`
or 
- All the Properties used in QAOA and RQAOA

Now in  `workflow_properties.py` there are:
- CircuitProperties (used in `qaoa_workflow.py` and `rqaoa_workflow.py`)
- ClassicalOptimizer (used in `baseworkflow.py`)
- BackendProperties (used in `baseworkflow.py`)

Reply:
I think having the current separation as it is now is fine since the `xxx_workflow_properties.py` contains objects that are specific to xxx and the `workflow_properties.py` in the root folder of algorithms can contain objects that are used in base AND objects that could be used in 1 or more algorithms.